### PR TITLE
Fix Gold Divisibility page

### PR DESCRIPTION
**Pull request by jt4517**
Some minor cleanups

### CODE REVIEW

File: content/4_Gold/Divis.mdx [hunk 1]
```diff
@@ -448,19 +448,10 @@ def gcd(a, b):
...
-$\gcd(F_n, F_{n + 1}) = \gcd(F_{n - 1}, F_n) = \dots = \gcd(0, F_1)$. This means
-that finding $\gcd(F_n, F_{n + 1})$ takes $n + 1$ steps, which is proportional
-to $\log \left(F_n F_{n+1}\right)$.
+This function runs in $\mathcal{O}(\log a + \log b)$ time because
+$a\le b \implies b \mod a < \frac{a + (b \mod a)}{2} < \frac{b}{2}$.
 
-<!-- (See [this Math Stack Exchange answer](https://math.stackexchange.com/questions/2477328/why-are-fibonacci-numbers-bad-for-euclids-algorithm-and-how-to-derive-this-uppe) -->
-<!-- ^ this doesn't seem very helpful -->
-<!-- Since $F_n = O(\varphi^n)$, the worst-case complexity of this algorithm is $\mathcal{O}(\log ab)$. -->
+The worst-case scenario for the Euclidean algorithm is when $a$ and $b$ are consecutive Fibonacci numbers $F_n$ and $F_{n + 1}$. In this case, the algorithm will calculate $\gcd(F_n, F_{n + 1}) = \gcd(F_{n - 1}, F_n) = \dots = \gcd(0, F_1)$. This means that finding $\gcd(F_n, F_{n + 1})$ takes $n + 1$ steps, which is proportional to $\log F_n + \log F_{n+1}$.
```
Comment (dongliu0426) [inline content/4_Gold/Divis.mdx:454]: What's wrong with the original explanation?

Comment (jt4517) [reply content/4_Gold/Divis.mdx:454]: There are small formatting mistakes and I think a few parts of the complexity analysis are unclear.

File: content/4_Gold/Divis.mdx [hunk 2]
```diff
@@ -502,7 +493,7 @@ $$
 
 ```cpp
 ll gcd(ll a, ll b){ return b == 0 ? a : gcd(b,a%b); }
-ll lcm(ll a, ll b) { return a/gcd(a,b)*b; }
+ll lcm(ll a, ll b)1 { return a/gcd(a,b)*b; }
```
Comment (dolphingarlic) [inline content/4_Gold/Divis.mdx:496]: ?

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

File: content/4_Gold/Divis.mdx (hunk 2 above)
Comment (jt4517) [reply content/4_Gold/Divis.mdx:496]: Yes, that shouldn't be there. I don't know what happened with it.

**Comment by bqi343** [comment]
Changes requested have not been made. Free free to create a new PR.

File: content/4_Gold/Divis.mdx (hunk 2 above)
Comment (dongliu0426) [reply content/4_Gold/Divis.mdx:496]:
```suggestion
ll lcm(ll a, ll b) { return a/gcd(a,b)*b; }
```

**Pull request closed without merging**